grommet/grommet-starter-new-app
# fix #4: har-validator error

#### What does this PR do?
Fixes #4 by updating the `package-lock.json` since there was a problem known issue in `har-validator 5.1.2`. 

#### How should this be manually tested?
* Run `npm install` locally with the changes in the `package-lock.json`
* Run `npm start` and app should run and visualized as expected.
